os/kernel/binary_manager/binary_manager_data.c

```diff
@@ -248,3 +251,35 @@ void binary_manager_remove_binlist(FAR struct tcb_s *tcb)
 		if (next) next->bin_blink = prev;
 	}
 }
+
+/****************************************************************************
+ * Name: binary_manager_clear_bindata
+ *
+ * Description:
+ *	 This function clears data of binary with bin_idx in a binary table.
+ *
+ ****************************************************************************/
+void binary_manager_clear_bindata(int bin_idx)
+{
+	struct tcb_s *tcb;
+
+	if (bin_idx < 0 || bin_idx > USER_BIN_COUNT) {
+		return;
+	}
+
+#ifdef CONFIG_OPTIMIZE_APP_RELOAD_TIME
+	tcb = sched_gettcb(BIN_ID(bin_idx));
+	/* If a 'reload' flag is true, a binary manager uses loading data, bininfo when loading the binary later.
```

Comment:
Could you please modify
a 'reload' flag -> 'reload' flag (since there is only 1 reload flag)
uses loading data, bininfo -> uses loading data 'bininfo' (loading data, bininfo suggests they are 2 separate entities)